pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [build-system]
 requires = [
-    "Cython~=0.29.29",
+    "cython~=3.0.2",
     "oldest-supported-numpy",
-    "setuptools>=61",
+    "setuptools>=67.8",
+    "wheel",
```

Comment:
I don't think `wheel` is an actual build requirement. It's necessary _if you're building wheels_, but not for building Fiona itself. `wheel` should only be installed wherever needs to build a wheel (and this is automatic for e.g., `cibuildwheel` or using `build` with the default build isolation).

Reply:
Correct, PyPA used to recommend this but it was removed a while back: https://github.com/pypa/packaging.python.org/pull/1050